install.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e
+
+GITHUB_SRC='git@github.com:valo/phoenix_with_nextjs.git'
+SKELETON_MODULE_NAME="PhoenixWithNextjs"
+SKELETON_PROJECT_NAME="phoenix_with_nextjs"
+
+NEW_PROJECT_NAME=$1
+NEW_MODULE_NAME=`elixir -e "Macro.camelize(\"$NEW_PROJECT_NAME\") |> IO.puts"`
+
+git clone $GITHUB_SRC
```

Comment:
You can specify the folder in which to clone the project. This way you won't need to rename the folder later.